src/theme/variables.css

```diff
@@ -76,6 +76,23 @@ http://ionicframework.com/docs/theming/ */
   --ion-color-light-tint: #f5f6f9;
 }
 
+.empty-state {
```

Comment:
Why are these added here?

Reply:
As sir, these styles need to be global as we can have multiple empty states in the app.

Reply:
yes, but I mean why on this section of the file?

Reply:
Sir, I thought these styles should be coded at last but before the media query, as the media query is written at last I think.

Reply:
Did you check where the other custom styles are? I would think they should be grouped together in some kind of sequence. Try that @sanskar345

Reply:
Okay sir done.